Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -87,18 +89,23 @@ try {
         }
     }
 
+    Write-Host "OUTPUTS:"
     $outSettingsJson = $outSettings | ConvertTo-Json -Depth 99 -Compress
-    Add-Content -Path $env:GITHUB_OUTPUT -Value "SettingsJson=$outSettingsJson"
-    Add-Content -Path $env:GITHUB_ENV -Value "Settings=$OutSettingsJson"
-    Write-Host "SettingsJson=$outSettingsJson"
+    if ($useBase64) {
+        Add-Content -Path $env:GITHUB_OUTPUT -Value "SettingsJson=$([Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($outSettingsJson)))"
```

Comment:
Readability: Move `Add-Content -Path $env:GITHUB_OUTPUT -Value "SettingsJson=$SettingsJsonValue"` outside of the if-else.

Reply:
fixed